### DISCUSSION TITLE
Would useConnect return the same instance when the config is the same?

### DISCUSSION THREAD

**Discussion by nguyenhuugiatri** [wevm/wagmi#1549] [Q&A]
I have 2 components:
1. Navbar
2. Content

both components are using useConnect's config
`const { connectors } = useConnect()`
`const { connect, error, isLoading } = useConnect({ connector: connectors[0] })`

My question: When I trigger the `connect` function in the component Navbar, will `error` and `isLoading` in the Content component return the same value?

**Comment by tmm** [comment]
Please add a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if you want folks to help you! It's hard to tell from the limited code you shared what could be going on.
